externalPlugins/EX-PO007-NamingConventions.js

```diff
@@ -66,9 +66,17 @@ var plugin = {
 var onPolicy = function(policy, cb) {
   var displayName = policy.getDisplayName(),
     policyType = policy.getType(),
-    prefixes = policyMetaData[policyType].indications,
     found = false,
-    hadWarn = false;
+    hadWarn = false,
+    policyMeta = policyMetaData[policyType];
+    
+  if (!policyMeta) {
+      // No prefix defined for this policy
+      found=false;
+      hadWarn=false;
+      return;
```

Comment:
We must  not return directly from  onPolicy without invoking the callback.  The convention is to check the cb to see if it is a function, then invoke it with the `hadWarn` flag either true or false as appropriate.  Also there's no need to set the booleans to the same values they already have.